ntcore/src/main/native/cpp/NetworkServer.cpp

```diff
@@ -221,10 +221,10 @@ void NetworkServer::ServerConnection4::ProcessWsUpgrade() {
       return;
     }
 
-    // TODO: set local flag appropriately
+    bool local = m_info.remote_ip == "127.0.0.1";
```

Comment:
Will this ever be "localhost"" instead? Or is this always an IP address?

Reply:
This is the IP the server sees from the socket, right?